toolset/setup/linux/languages/rust.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+RETCODE=$(fw_exists $IROOT/rust.installed)
+[ ! "$RETCODE" == 0 ] || { . $IROOT/rust.installed; return 0; }
+
+fw_get https://static.rust-lang.org/dist/rust-1.0.0-x86_64-unknown-linux-gnu.tar.gz
+fw_untar rust-1.0.0-x86_64-unknown-linux-gnu.tar.gz
+(
+	cd rust-1.0.0-x86_64-unknown-linux-gnu
+	./install.sh --prefix=$IROOT/rust
+)
+
+echo "export LD_LIBRARY_PATH=\${LD_LIBRARY_PATH}:\$IROOT/rust/lib" >> $IROOT/rust.installed
+echo "export PATH=\$IROOT/rust/bin:$PATH" >> $IROOT/rust.installed
```

Comment:
I think you have your echo statements a bit messed up. I would expect:

`echo -e "export LD_LIBRARY_PATH=${IROOT}/rust/lib:\$LD_LIBRARY_PATH" > $IROOT/rust.installed
echo -e "export PATH=${IROOT}/rust/bin:\$PATH" >> $IROOT/rust.installed`

The `-e` flag allows escaping the dollar sign, and otherwise it doesn't matter if $IROOT gets expanded at this point, so save a char.